src/dafny/FileIO/FileIO.md

```diff
@@ -0,0 +1,122 @@
+
+## The `FileIO` module {#sec-fileio}
+
+_The FileIO methods currently only work with `--unicode-char:false`, which is not the default for Dafny 4._
```

Comment:
Good callout, and we should cut an issue to track this.

Reply:
There already is such an issue: #90